day_two_guide/topics/etcd_backup.adoc

```diff
@@ -0,0 +1,1167 @@
+
+Module included in the following assemblies:
+
+* day_two_guide/host_level_tasks.adoc
+////
+
+etcd is the key value store for all object definitions, as well as the
+persistent master state. Other components watch for changes, then bring
+themselves into the desired state.
+
+{product-title} versions prior to 3.5 used etcd version 2, while 3.5 and later
+use v3. The data model between the two versions of etcd is different. etcd v3
+can use both the v2 and v3 data model, whereas etcd v2 can only use the v2 data
+model. In an etcd v3 server, the v2 and v3 data stores exist in parallel and
+are independent.
+
+For both v2 and v3 operations, you can use the `ETCDCTL_API` environment
+variable to use the proper API:
+
+----
+$ etcdctl -v
+etcdctl version: 3.2.5
+API version: 2
+$ ETCDCTL_API=3 etcdctl version
+etcdctl version: 3.2.5
+API version: 3.2
+----
+
+See xref:../install_config/upgrading/migrating_etcd.adoc[Migrating etcd Data (v2
+to v3) section] for information about how to migrate to v3.
+
+=== Backup and restore etcd
+
+The etcd backup process is composed of two different procedures:
+
+* Configuration backup: Including the required etcd configuration and
+certificates
+* Data backup: Including both v2 and v3 data model.
+
+The data backup procedure can be done on any host that has connectivity to the
+etcd cluster, where the proper certificates are provided, and where the
+`etcdctl` tool is installed.
+
+[NOTE]
+====
+The backup files must be copied to an external system other than the etcd
+members themselves, ideally outside the {product-title} environment, and those
+files encrypted.
+====
+
+==== etcd configuration backup
+
+The etcd configuration files to be preserved are all stored in the `/etc/etcd`
+directory of the instances where etcd is running. This includes the etcd
+configuration file (`/etc/etcd/etcd.conf`) and the required certificates for
+cluster communication. All those files are generated at installation time by the
+Ansible installer.
+
+To back up the etcd configuration: 
+
+----
+$ ssh master-0
+# mkdir -p /backup/etcd-config-$(date +%Y%m%d)/
+# cp -R /etc/etcd/ /backup/etcd-config-$(date +%Y%m%d)/
+----
+
+[NOTE]
+====
+The backup is to be performed on every etcd member of the cluster
+as the certificates and configuration files are unique.
+====
+
+==== etcd configuration restore
+
+The restore procedure for etcd configuration files replaces the appropriate
+files, then restarts the service.
+
+If a etcd host has become corrupted and the `/etc/etcd/etcd.conf` file is lost,
+restore it using:
+
+----
+$ ssh master-0
+# cp /backup/yesterday/master-0-files/etcd.conf /etc/etcd/etcd.conf
+# restorecon -Rv /etc/etcd/etcd.conf
+# systemctl restart etcd.service
+----
+
+In this example, the backup file is stored in the
+`/backup/yesterday/master-0-files/etcd.conf` path where it can be used as an
+external NFS share, S3 bucket, etc.
+
+[[etcd-data-backup]]
+==== etcd data backup
+
+[discrete]
+== Prerequisites
```

Comment:
I think the heading level is wrong here. If it's nested under the etcd data backup then it should be =====